src/state/slices/opportunitiesSlice/selectors/combined.ts

```diff
@@ -96,17 +96,43 @@ export const selectAggregatedEarnOpportunitiesByAssetId = createDeepEqualOutputS
     const combined = [...userStakingOpportunites, ...userLpOpportunities]
     const totalFiatAmountByAssetId: Record<AssetId, BN> = {}
     const projectedAnnualizedYieldByAssetId: Record<AssetId, BN> = {}
+
+    // TODO(gomes): make me a selector?
```

Comment:
We already have `selectIsActiveStakingOpportunityByFilter`, might want a `selectIsActiveOpportunityByFilter` counterpart selecting combined opportunities

Reply:
Actually not sure about the `byFilter` selector - `byAssetId` / `byProvider` and `byOpportunity` are all quite different